pkg/controller/daemon/update.go

```diff
@@ -88,6 +88,7 @@ func (dsc *DaemonSetsController) constructHistory(ds *apps.DaemonSet) (cur *apps
 	if err != nil {
 		return nil, nil, err
 	}
+	max := int64(0)
```

Comment:
why int64?

Reply:
No particular reason - just to be on the safe side.